### PR TITLE
Rely on sbt-slamdata

### PR DESCRIPTION
**Pull request by drostron**
Depends on slamdata/sbt-slamdata#1

### CODE REVIEW

File: src/main/scala/slamdata/Predef.scala [hunk 1]
```diff
@@ -14,67 +14,81 @@
...
+  type Vector[+A]                = scala.Vector[A]
+  val  Vector: scala.Vector.type = scala.Vector
+
+  type BigDecimal                    = M.BigDecimal
+  val  BigDecimal: M.BigDecimal.type = M.BigDecimal
+
+  type BigInt                = M.BigInt
+  val  BigInt: M.BigInt.type = M.BigInt
 
   type Iterable[+A] = C.Iterable[A]
```
Comment (sellout) [inline src/main/scala/slamdata/Predef.scala]: Get rid of this one.

File: src/main/scala/slamdata/pkg/PackageAliases.scala [hunk 2]
```diff
@@ -14,10 +14,12 @@
...
-package quasar.pkg
+package slamdata.pkg
 
-import scala.collection.{ mutable => scm, immutable => sci }
 import java.lang.SuppressWarnings
+import scala.{collection => sc}
+import scala.collection.{ mutable => scm, immutable => sci }
+import scala.{util => su}
 
 @SuppressWarnings(scala.Array("org.wartremover.warts.MutableDataStructures"))
```
Comment (sellout) [inline src/main/scala/slamdata/pkg/PackageAliases.scala]: If you check out the edits for this file (and PackageMethods) in https://github.com/quasar-analytics/quasar/compare/master...sellout:sql-distinct#diff-91eff94d671cb9207858b1d3f4d061d0 you can get rid of this Suppression and a bunch of other things that shouldn’t be in our Predef.

**Review by sellout — CHANGES_REQUESTED** [review]
Some cleanup to the Predef, and otherwise just need to get the other PR merged so this can pass its build.

File: src/main/scala/slamdata/pkg/PackageAliases.scala (hunk 2 above)
Comment (drostron) [reply src/main/scala/slamdata/pkg/PackageAliases.scala]: Excellent, will ✄ accordingly.

**Comment by drostron** [comment]
@sellout Ready for another look

**Review by sellout — APPROVED** [review]

**Pull request merged**